src/include/storage/store/var_list_column_chunk.h

```diff
@@ -67,6 +66,9 @@ class VarListColumnChunk : public ColumnChunk {
     inline common::offset_t getListOffset(common::offset_t offset) const {
         return offset == 0 ? 0 : getValue<uint64_t>(offset - 1);
     }
+
+private:
```

Comment:
Redundant access specifier.

Reply:
We always put an access specifier for functions and fields.